### PR TITLE
[Merged by Bors] - chore(*): remove empty lines between variable statements

### PR DESCRIPTION
**Pull request by jcommelin**
Empty lines were removed by executing the following Python script twice
```python
import os
import re


# Loop through each file in the repository
for dir_path, dirs, files in os.walk('.'):
  for filename in files:
    if filename.endswith('.lean'):
      file_path = os.path.join(dir_path, filename)

      # Open the file and read its contents
      with open(file_path, 'r') as file:
        content = file.read()

      # Use a regular expression to replace sequences of "variable" lines separated by empty lines
      # with sequences without empty lines
      modified_content = re.sub(r'(variable.*\n)\n(variable(?! .* in))', r'\1\2', content)

      # Write the modified content back to the file
      with open(file_path, 'w') as file:
        file.write(modified_content)
```

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Archive/Hairer.lean [hunk 1]
```diff
@@ -32,7 +32,6 @@ variable {ι : Type*} [Fintype ι]
 section normed
 variable {𝕜 E F : Type*} [NontriviallyNormedField 𝕜]
 variable [NormedAddCommGroup E] [NormedSpace 𝕜 E] [NormedAddCommGroup F] [NormedSpace 𝕜 F]
-
```
Comment (MichaelStollBayreuth) [inline Archive/Hairer.lean:35]: I think this is a case where the empty line makes sense, as the next line is actually `variable ... in`, which applies specifically to the following declaration and should not be put together with the global `variable` lines.

**Comment by MichaelStollBayreuth** [comment]
I object to the `easy` tag; the time to look through all of this is definitely more than 20 seconds!

**Comment by MichaelStollBayreuth** [comment]
In general, I have no strong feelings here.

A thought: It may make sense to separate various bits of `variable` declarations also optically when they correspond to a natural grouping. But perhaps in this case, it would make sense to add comments.

There may be false positives like the very first change, where the line after the empty ony is actually `variable ... in`. This specific case should be easy to fix, but there may be other cases.

**Comment by sgouezel** [comment]
I agree that the `variable ... in` would need special treatment. @jcommelin, what do you think?

**Comment by jcommelin** [comment]
Fair enough! Thanks a lot for spotting this. I think I should adapt the script, and regenerate a diff.

File: Archive/Hairer.lean (hunk 1 above)
Comment (jcommelin) [reply Archive/Hairer.lean]: This is now fixed

**Comment by adomani** [comment]
If this is an actual style recommendation, should the script be added to the linting/reviewdog to prevent future lapses?

**Comment by sgouezel** [comment]
bors r+
Thanks!

**Comment by riccardobrasca** [comment]
Thanks!

bors d+

**Pull request closed without merging**